check-smtp/lib/check-smtp.go

```diff
@@ -0,0 +1,125 @@
+package checksmtp
+
+import (
+	"os"
+
+	"net/smtp"
+
+	"fmt"
+
+	"crypto/tls"
+
+	"net"
+
+	"time"
```

Comment:
I noticed too many new lines here.

Reply:
Thanks, I fix import statement in https://github.com/mackerelio/go-check-plugins/pull/243/commits/30249cdb7cb9d033d93ba15e20b138bff8a96a16.